docs/flow-visualization.md

```diff
@@ -0,0 +1,72 @@
+# Flow Visualization
+## Purpose
+Antrea supports sending IPFIX flow records as flow exporter. The Elastic Stack
+(ELK Stack) works
+as a data collector for flow records and flow-related information can be
+visualized
+in Kibana. This doc provides guidelines for deploying Elastic Stack with
+support for
+Antrea-specific fields in a Kubenetes cluster.
+
+## About Elastic Stack
+[Elastic Stack](https://www.elastic.co) is a group of open source products from
+Elastic to help collect, store, search, analyze and visualize data in real
+time. We will use Logstash, Elasticsearch and Kibana in Antrea flow
+visualization.
+[Logstash](https://www.elastic.co/logstash) works as data collector to
+centralize flow records. [Logstach Netflow codec
+plugin](https://www.elastic.co/guide/en/logstash/current/plugins-codecs-netflow.html)
+supports Netflow v5/v9, sFlow and IPFIX protocols for flow data collection.
```

Comment:
Can Netflow plugin supports sFlow protocol? Please mention Netflow v10 in parenthesis for IPFIX so there won't be any confusion with plugin name.

Reply:
Thanks for pointing this out. I think it only supports v5/v9/v10